egs/ami/s5b/local/rnnlm/tuning/run_lstm_tdnn_a.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+
+# Copyright 2012  Johns Hopkins University (author: Daniel Povey)  Tony Robinson
+#           2017  Hainan Xu
+#           2017  Ke Li
+
+# 2-layer LSTM hidden 200:
+# rnnlm/train_rnnlm.sh: best iteration (out of 18) was 10, linking it to final iteration.
```

Comment:
Please modify train_rnnlm.sh so that after this line, it prints out something like:
```
  train/dev perplexity was -83.7/-128.4.
```
These should be extracted from the compute_prob_{train,valid} logs of the 'best' iteration, the one that we linked to the final iteration,  and exponentiated.  If you have to write a perl or python script to do this, then do it (although a simple inline awk script would probably suffice).